lisa/tools/git.py

```diff
@@ -153,3 +153,12 @@ def init_submodules(self, cwd: pathlib.PurePath) -> None:
             expected_exit_code=0,
             expected_exit_code_failure_message="error on init submodules.",
         )
+
+    def get_latest_tag(self, cwd: pathlib.PurePath) -> str:
+        # sort tags by version, grab the last in the list
+        return self.run(
+            "--no-pager tag --sort=v:refname",
```

Comment:
the "latest" may be different for each repo. How about expose two arguments, and rename it. So it can return a couple of ordered tags.

`def get_tag(self, cwd: pathlib.PurePath, sort_by:str = "v:refname", filter:str = "", return_first: bool = True) -> str`